multichain-testing/test/support.ts

```diff
@@ -9,28 +9,26 @@ import { makeGetFile, makeSetupRegistry } from '../tools/registry.js';
 import { generateMnemonic } from '../tools/wallet.js';
 import { makeRetryUntilCondition } from '../tools/sleep.js';
 import { makeDeployBuilder } from '../tools/deploy.js';
+import chainInfo from '../starship-chain-info.js';
 
 const setupRegistry = makeSetupRegistry(makeGetFile({ dirname, join }));
 
-export const chainConfig = {
+export type ChainName = 'agoric' | 'cosmoshub' | 'osmosis';
```

Comment:
very confusing to have `ChainName` be a literal for these three. what's the motivation?

Reply:
Thanks for pairing on this earlier. This is now removed